NewEvaluation.html

```diff
@@ -654,20 +643,18 @@
 			</fieldset>
 		</form>
 
-	<div class="littleSpacing"></div>
-
-	<form  action="Evaluation.html">
-		<input  type="submit" name="button" value="Submit">
-	</form>
+		<div class= "submit-button">
+			<form action="Evaluation.html">
+				<input  type="submit" name="button" value="Submit">
```

Comment:
Yup, that's more or less what I'm suggesting. Not the data processing part just yet, naturally (since we've just started the JS basics), but just the markup boilerplate that will allow us to pass the appropriate data to the backend.